docs/source/testing/mocking-responses.mdx

```diff
@@ -0,0 +1,181 @@
+---
+title: Mocking responses
+---
+
+Apollo Kotlin provides two tools to mock responses in tests:
```

Comment:
If this is too long we can also split in 2 pages (with cross references). LMK what you think :)

Reply:
Yep, I'd be in favour of splitting.